arbitrum-docs/partials/_migrate-nft-collections-partial.md

```diff
@@ -0,0 +1,90 @@
+
+The [Arbitrum token bridge](https://bridge.arbitrum.io/) is a trustless, permissionless bridge that allows users to transfer **ERC-20 tokens** (not ERC-721 or ERC-1115 tokens) between Ethereum and <a data-quicklook-from="arbitrum-one">Arbitrum One</a> and <a data-quicklook-from="arbitrum-nova">Nova</a> chains. To use the bridge, visit the [Arbitrum Bridge portal](https://bridge.arbitrum.io/) and follow the instructions.
+
+
+## Third-party NFT bridging solutions
+
+This option is suitable for **end-users** and **developers** who want to migrate **non-ERC-20 tokens** (e.g. ERC-721 or ERC-1155 tokens) to one of Arbitrum's chains. There are many things to consider before using a third-party bridging solution, including:
+
+ - Token metadata
+ - Royalty mechanisms
+ - Perception of which chain's NFTs are canonical
+ - Representational clashes if multiple bridges are used for the same collection
+ - etc
+
+Using a third-party bridging solution may introduce new trust assumptions, forcing you to compromise with properties of permissionlessness and/or trustlessness. Consider these tradeoffs carefully when deciding to use any of the below solutions, and proceed with caution. This list is presented in alphabetical order, and does not represent an endorsement or recommendation; it's simply a list of options that are available, shared with the expectation that you'll **do your own research before using them**:
+
+- [Axelar](https://axelar.network/)
+    - Currently live on Arbitrum One for ERC721 and ERC1155
+- [Layer Zero](https://layerzero.network/)
+    - Currently live on Arbitrum One for ERC721 and ERC1155
+- [Wormhole](https://wormhole.com/)
+    - Currently live on Arbitrum One for ERC721
+- [XP Network](https://xp.network/)
+    - Currently live on Arbitrum Nova for ERC721 and ERC1155
+
+
+### Other third-party solutions
+
+There are some solutions that give you the best of both worlds: **canonical representation of foundational NFTs on their original chain**, *and* **Arbitrum's scaling capabilities to facilitate fast and cheap experiences**:
+
+ - **Pirate Nation** has implemented a solution they coined as **[mirroring](https://piratenation.medium.com/mirroring-trade-on-l1-play-on-l2-3f53068a6a44)**, which allows users to trade NFTs on Ethereum and play with the mirrored versions of those NFTs on Arbitrum Nova. The NFTs on Nova's L2 chain can be transferred by a custodial central entity, but the NFTs on L1 remain fully self-custodial. This solution entirely avoids the need for bridging.
+ - **0xEssential**'s [NFT Global Entry](https://docs.0xessential.com/0xessential/guides/nft-global-entry) offers a similar solution by hosting a public oracle that generates ownership proofs; these proofs can be easily consumed and used by their SDK.
+
+
+## Do-it-yourself (DIY) bridging
```

Comment:
@amarrazza - low-confidence about this reframing - would love to know how this feels

Reply:
@TucksonDev I mentioned that option 1 and 4 are similar here too :) the solution is likely renaming the bucket to "use a third-party bridge" now that the scope has been changed